lib/performance_data/statistics.rb

```diff
@@ -0,0 +1,216 @@
+require 'json'
+require 'gds_api/performance_platform/data_out'
+
+module PerformanceData
+  class Statistics
+    attr_accessor :data_out, :content, :slug, :part_urls, :document_type, :is_multipart, :performance_data
+
+    def initialize(content, slug)
+      @data_out = GdsApi::PerformancePlatform::DataOut.new("http://www.performance.service.gov.uk")
+      @content = content
+      @slug = slug
+      @part_urls = get_part_urls(content, slug)
+      @document_type = @content["document_type"] || @content["format"]
+      @is_multipart = @part_urls.any? || (document_type == 'smart-answer')
```

Comment:
Actually, there is something we can do.  Content-schemas has [a definition for parts](https://github.com/alphagov/govuk-content-schemas/blob/master/formats/definitions.json#L228-L251) and `web_url` is no longer a valid key for parts so we can drop the bit of code that looks for `web_url` in the parts.  Probably as a separate story along side other tidying up (add it to the one you already wrote?)